src/main/java/com/eternalcode/discordapp/database/model/User.java

```diff
@@ -0,0 +1,34 @@
+package com.eternalcode.discordapp.database.model;
+
+import com.j256.ormlite.field.DatabaseField;
+import com.j256.ormlite.table.DatabaseTable;
+
+@DatabaseTable(tableName = "users")
+public class User {
+
+    @DatabaseField(id = true)
+    private long id;
+
+    @DatabaseField
+    private String username;
+
+    public User() {
+    }
+
+    public User(String username) {
+        this.username = username;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+}
```

Comment:
Z moich doświadczeń z ormlitem, to zawsze miałem zwykły obiekt User bez tych adnotacji od ormlite'a, później tworzyłem Wrapper do takiego User'a. Proponuję przemyślenie tego.

Reply:
No to musi być w innej klasie która wrappuje usera